DeAceroDLabs/marketplace-frontend
# Fix/dmp 536 Dropdown icon on field

## What is done

[[DMP-536] Dropdown icon on field](https://deacero.atlassian.net/browse/DMP-536)

## How is done

- Added `ArrowDropDownIcon` icon to `Dropdown`  component.

## How to test

1. Run the project via `npm start`.
2. /signup
3. Fill first form
4. You should see the dropdown icon on the last form on Colonia Field.
